internal/pkg/bulk/bulk_test.go

```diff
@@ -255,8 +254,8 @@ func TestCancelCtx(t *testing.T) {
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
 			ctx, cancelF := context.WithCancel(context.Background())
-			ctx = testlog.SetLogger(t).WithContext(ctx)
```

Comment:
Had to remove the test loggers from these tests as they were still being used after the test case terminated